### DISCUSSION TITLE
How to verify a specific version?

### DISCUSSION THREAD

**Discussion by evehash** [janoside/btc-rpc-explorer#505] [Q&A]
Hi janoside,

I'm working on a project to deploy automatically a bitcoin node with docker and I'm missing a way to verify the repo.

I was trying to verify locally the tag v3.3.0 but it seems is not possible, the tag was not included... and cosidering that the release files (zip and tar.gz) are created by Github, I think there is no way to verify the signatures... or I'm wrong?

This is why a get after importing your public key from keybase.
```
~/Desktop/btc-rpc-explorer$ git verify-tag v3.3.0
error: v3.3.0: cannot verify a non-tag object of type commit.
```

Please, can you provide a way to verify the repo? Ty!

**Comment by janoside** [comment]
@reverse-hash I've been using "lightweight" git tags rather than "annotated" tags. The command you posted only works for annotated tags. Sorry about this. If you want to see signatures for various commits, you can do so with `git log --show-signature`.

**Comment by Shortyd51** [comment]
Thank you I'm still learning
